libs/deal-ticket/src/components/deal-ticket/deal-ticket-stop-order.tsx

```diff
@@ -181,7 +189,7 @@ const Trigger = ({
                       data-testid={`triggerPrice${oco ? '-oco' : ''}`}
                       type="number"
                       step={priceStep}
-                      appendElement={assetSymbol}
+                      appendElement={<Pill size="xs">{quoteName}</Pill>}
```

Comment:
The padding of the pill and the alignment of it within the input looks off to me. This can be fixed later but if its a quick win lets just do it. We should:

- Reduce the py of the pill
- Reduce the padding right within the input when the pill is present
- 
![Screenshot 2023-08-29 at 21 16 28](https://github.com/vegaprotocol/frontend-monorepo/assets/6803987/80f4f7a6-5af5-4c47-82f6-ffce9524094c)